app.js

```diff
@@ -8,7 +8,11 @@ app.get('/api', getWelcomeMessage)
 
 app.get('/api/topics', getTopics)
 
-//app.get('/api/articles', getArticles)
+app.get('/api/articles', getArticles)
+
+app.use((req, res, next) => {
```

Comment:
nice  to see some error handling here ... be mindful of how this will behave as you work on later tickets